app/spike/internal/cmd/secret/new.go

```diff
@@ -18,6 +20,12 @@ func NewSecretCommand(
 	cmd := &cobra.Command{
 		Use:   "secret",
 		Short: "Manage secrets",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+        if lock.IsLocked() { 
+            return fmt.Errorf("SPIKE is locked — please unlock before running this command")
+        }
+        return nil
```

Comment:
Do not check for locks inside SPIKE Pilot.

Lock checks will be done within SPIKE Nexus, and we should not be using a lock file for this.